app/src/components/buttons/NavbarItemLinkButton.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { Button } from 'src/components/base';
+import { pxToRem, colors } from 'src/styled-theme';
+
+export interface NavbarItemLinkButtonProps {
+  readonly title: string;
+  readonly children: React.ReactNode;
+
+  readonly selectedRoute: any;
+
+  readonly handleNavItemSelection: (event: {
```

Comment:
If we simply pass in the `key` as per my below comment, we should be able to simplify the types here.

Reply:
+1